src/standalone/DataGrid/Content/Cell.tsx

```diff
@@ -82,19 +83,19 @@ const Cell = (props: CellProps): React.ReactElement => {
 			onMouseLeave={endHover}
 			onClick={toggleSelection}
 			onDoubleClick={editRecord}
-			className={`${classes.cell} ${
-				props.rowIndex !== 0 ? classes.dataCell : classes.headerCell
-			} ${
+			className={combineClassNames([
+				classes.cell,
+				props.rowIndex !== 0 ? classes.dataCell : classes.headerCell,
 				props.rowIndex !== 0
 					? props.rowIndex % 2 === 0
 						? classes.rowEven
 						: classes.rowOdd
-					: ""
-			} ${props.rowIndex !== 0 && column ? `column-${column.field} ` : ""}${
+					: "",
+				props.rowIndex !== 0 && column ? `column-${column.field} ` : "",
```

Comment:
same here, also this one contains an extra space which should be removed
`props.rowIndex !== 0 && column && `column-${column.field}`

Reply:
@TQjona , updated changes. Please Review.